web/src/app/shared/models/help.model.ts

```diff
@@ -0,0 +1,67 @@
+import { Help } from '@app/main/components/help/help';
+
+export class HelpModel {
+  icon: string;
+  title: string;
+  content: string;
+  topics: Help[] = [
```

Comment:
I know this was here before, but `Help` should be a model, may be `HelpTopic`?

Reply:
okay